src/resources/LibraryStyles.css

```diff
@@ -397,7 +398,6 @@ input {
 .SearchResultItemContainer .ItemIcon {
     padding: 2px 8px;
     width: 40px;
-    height: 40px;
```

Comment:
Hi @fanwgwg,

Seems like the problem with distorted icons here is caused by the fact that `.ItemInfo` on the right side of the icon space is set to match the width of its parent element (`width: 100%`), hence pushing (and compressing) the icon to the left. Could you please remove the `width: 100%` of `.ItemInfo` instead and leave the `.ItemIcon` with its native dimensions at 32 × 32px? Thanks!